### PR TITLE
Info plugin: add option to print only keys

### PR DESCRIPTION
**Pull request by GuilhermeHideki**
Tries to solve one part of #544: list attributes of the items from the query.
Not sure how to better the help message, needs documentation too.

### CODE REVIEW

**Comment by sampsyo** [comment]
This seems like a good direction. I'm still not 100% clear on the use case---and if you really need a list of keys, can't you just pipe the output to `cut -d: -f1`? But if this will be useful, I don't object to merging (although we'll need those docs updated first).

Thanks for looking into it!

File: beetsplug/info.py [hunk 1]
```diff
@@ -119,6 +119,24 @@ def print_data(data, item=None, fmt=None):
...
+    formatted = []
+    for key, value in data.iteritems():
+        formatted.append(key)
+
+    if len(formatted) == 0:
+        return
+
+    line_format = u'{0}{{0}}'.format(u' ' * 4)
+    if path:
+        ui.print_(displayable_path(path))
```
Comment (GuilhermeHideki) [inline beetsplug/info.py]: The problem with the `cut -d: -f1` is when the item's path is printed, like d:\music

**Comment by GuilhermeHideki** [comment]
**tl;dr** I just implemented because seems easy and the issue was open

my use case was to search the fields for some custom names (to keep names normalized like track vs track#), so I searched the issues and found that issue open. this doesn't help 100% for me, but at least doesn't make walls of text just to see some fields

File: beetsplug/info.py (hunk 1 above)
Comment (sampsyo) [inline beetsplug/info.py]: Ah, I see—so it's useful if you're printing multiple files then.

**Comment by sampsyo** [comment]
Great; thanks! Merged.
